src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx

```diff
@@ -133,14 +133,14 @@ const FlightWidget = (props: FlightWidgetProps) => {
                             {' '}
                             FROM SIMBRIEF
                         </button>
-                        <button
-                            type="button"
-                            className="ml-1 w-1/2 text-white bg-green-500 p-2 flex items-center justify-center rounded-lg focus:outline-none opacity-50 text-lg"
-                        >
-                            <IconLink className="mr-2" size={23} stroke={1.5} strokeLinejoin="miter" />
-                            {' '}
-                            LINK MCDU
-                        </button>
+                        {/*<button*/}
+                        {/*    type="button"*/}
+                        {/*    className="ml-1 w-1/2 text-white bg-green-500 p-2 flex items-center justify-center rounded-lg focus:outline-none opacity-50 text-lg"*/}
+                        {/*>*/}
+                        {/*    <IconLink className="mr-2" size={23} stroke={1.5} strokeLinejoin="miter" />*/}
+                        {/*    {' '}*/}
+                        {/*    LINK MCDU*/}
+                        {/*</button>*/}
```

Comment:
```suggestion

```
Hi man! Just get rid of it. :see_no_evil:

Reply:
Will do :D